doc/tut3.rst

```diff
@@ -324,14 +342,14 @@ possible with it.
 Strformat
 ---------
 
-In the Nim standard library, the ``strformat`` library provides a
+In the Nim standard library, the `strformat` library provides a
```

Comment:
unrelated to this PR, and won't render properly in github until you add `.. default-role:: code` as described in https://github.com/nim-lang/Nim/pull/17028

PR welcome for changing double backticks to single backticks in rst files in a dedicated PR(s) (you can batch them as needed, preferably manual.rst should be its own PR as it's large) using the same approach as https://github.com/nim-lang/Nim/pull/17028, I can review it fast.

Reply:
> I can review it fast

unfortunately, https://github.com/nim-lang/Nim/issues/17260 happened